LSPS6/README.md

```diff
@@ -0,0 +1,75 @@
+# LSPS6 LSP meta-data
+
+| Name    | `meta-data` |
+| ------- |---------------|
+| Version | 1             |
+| Status  | Draft         |
+
+LSPS6 requires [BOLT8][] as a transport layer.
+
+[BOLT8]: https://github.com/lightning/bolts/blob/master/08-transport.md
+
+## Motivation
+
+Not all information is available through regular LSPS messages.
+In order to provide more transparency, LSPS6 provides the ability to deliver extra information.
+This extra information can be in the form of a URL, icon, logo and/or a support email address.
+We need to be conscious about information spoofing.
+
+### Actors
+
+The 'LSP' is the API provider, and acts as the server.
+The 'client' is the API consumer.
+
+## DNS Public Key Verification
+
+[BIP353][] provides a way to discover payment instructions over DNS.
+Analogous to this the LSP SHOULD create a DNS TXT records in the form of
+`lsps.acme.com. 3600 IN TXT "public_key:0326e6...2833a3"` for public key verification.
```

Comment:
```suggestion
`lsps6.acme.com. 3600 IN TXT "public_key:0326e6...2833a3"` for public key verification.
```

Should we use `lsps6` as a name here in cast an other lsps uses DNS entries?

Reply:
Please us the full node id in this example and not `...` to make the spec crystal clear.